Scene_Score.pde

```diff
@@ -32,16 +33,27 @@ class Scene_Score implements Scene {
     text("Name:", width/4, height*0.5);
 
     textSize(128);
-    text(textInputted, width/4, height*0.6);
-
-    if (textInputting) {
-      if (millis() % 2000 < 1000) {
-        float x = width/4 + textWidth(textInputted)/2;
-        stroke(255);
-        strokeWeight(2);
-        line(x, height*0.59, x, height*0.73);
+    rectMode(CENTER);
+    noStroke();
+    float charWidth = 80;
+    for (int i = 0; i < 3; i++) {
+      float x = width/4 - charWidth + charWidth * i;
```

Comment:
Could you add some brackets here?
To better indicate the order this formula follows.